src/lib/address/address.ts

```diff
@@ -39,7 +42,7 @@ export const parseMaybeUrlWithChainId = ({
         const parsedUrl = parseEthUrl(urlOrAddress)
 
         if (parsedUrl.parameters?.address) {
-          throw new Error('QR codes with address parameter are not supported')
+          throw new Error(DANGEROUS_ETH_URL_ERROR)
```

Comment:
Kind of weird throwing, then re-throwing it 2 times but has to be because of our exhaustion logic, this is an *akschual* errors as opposed to exhaustion expected, swallowed errors 

![chomp](https://media.tenor.com/QrtakO85U3UAAAAC/brown-chewing.gif)